res/openvpn-gui-res-no.rc

```diff
@@ -371,29 +370,29 @@ BEGIN
     IDS_ERR_SOCKS_PROXY_PORT "SOCKS proxy port må oppgis."
     IDS_ERR_SOCKS_PROXY_PORT_RANGE "SOCKS proxy port må være ett tall mellom 1 og 65535"
     IDS_ERR_CREATE_REG_HKCU_KEY "En feil oppsto ved opprettelse av register-nøkkel ""HKEY_CURRENT_USER\\%s"""
-    IDS_ERR_GET_TEMP_PATH "En feil oppsto når GetTempPath() ble kalt opp. Alternativ ""C:\\"" vil bi benyttet."
+    IDS_ERR_GET_TEMP_PATH "En feil oppsto når GetTempPath() ble kalt opp. Alternativ ""C:\\"" vil bli benyttet."
 
     /* service */
     IDS_ERR_OPEN_SCMGR_ADMIN "En feil oppsto under åpning av Service Manager. Du har sannsynligvis ikke rettigheter til å starte/stoppe en service."
-    IDS_ERR_OPEN_VPN_SERVICE "Kunnne ike åpne ""OpenVPNService"""
+    IDS_ERR_OPEN_VPN_SERVICE "Kunnne ikke åpne ""OpenVPNService"""
     IDS_ERR_START_SERVICE "En feil oppsto under oppstart av OpenVPN Service"
-    IDS_ERR_QUERY_SERVICE "En feil oppsto når status på OpenVPN Service skulle kontrolleres."
+    IDS_ERR_QUERY_SERVICE "En feil oppsto da status på OpenVPN Service skulle kontrolleres."
     IDS_ERR_SERVICE_START_FAILED "OpenVPN Service kunne ikke starte."
     IDS_ERR_OPEN_SCMGR "OpenSCManager feilet (%d)"
     IDS_ERR_STOP_SERVICE "En feil oppsto ved stopp av OpenVPN Service"
     IDS_NFO_RESTARTED "OpenVPN Service omstart utført."
 
     /* registry */
     IDS_ERR_GET_WINDOWS_DIR "Kunne ikke finne katalog hvor Windows er installert."
-    IDS_ERR_GET_PROGRAM_DIR "Kunne ikke finne path til din ""Program"" katalog."
+    IDS_ERR_GET_PROGRAM_DIR "Kunne ikke finne filstien til din ""Program"" katalog."
```

Comment:
"mappe" should probably be used instead of "katalog" to be consistent with the terminology used earlier.